### PR TITLE
Add bullet and height map metadata

### PR DESCRIPTION
**Pull request by scpeters**
Port of https://github.com/ignition-release/ign-physics4-release/pull/9 and https://github.com/ignition-release/ign-physics4-release/pull/12 to `ignition-physics5`

### CODE REVIEW

**Comment by scpeters** [comment]
testing: [Build Status](https://build.osrfoundation.org/job/ign-physics5-debbuilder/215/) https://build.osrfoundation.org/job/ign-physics5-debbuilder/215/

File: focal/debian/control [hunk 1]
```diff
@@ -186,16 +203,52 @@ Description: Ignition Physics classes and functions for robot apps - TPE library
...
-         libignition-physics5-tpelib-dev,
+Depends: libignition-physics5-core-dev (= ${binary:Version}),
+         libignition-physics5-dartsim-dev (= ${binary:Version}),
+         libignition-physics5-heightmap-dev (= ${binary:Version}),
+         libignition-physics5-mesh-dev (= ${binary:Version}),
+         libignition-physics5-sdf-dev (= ${binary:Version}),
+         libignition-physics5-tpe-dev (= ${binary:Version}),
+         libignition-physics5-tpelib-dev (= ${binary:Version}),
          ${misc:Depends}
+# Bullet not added on purpose to avoid adding a new dependency to a stable release
```
Comment (chapulina) [inline focal/debian/control]: We can add it here, since it's not stable yet, what do you think?

Comment (scpeters) [reply focal/debian/control]: ok sure

Comment (scpeters) [reply focal/debian/control]: https://github.com/ignition-release/ign-physics5-release/pull/2/commits/2362187ada5fdb127ca59262c838b7382d82f246

**Review by chapulina — APPROVED** [review]

**Pull request merged**